localstack/services/awslambda/lambda_executors.py

```diff
@@ -717,6 +718,13 @@ def get_host_path_for_path_in_docker(cls, path):
         return re.sub(r'^%s/(.*)$' % config.TMP_FOLDER,
                       r'%s/\1' % config.HOST_TMP_FOLDER, path)
 
+    @classmethod
+    def format_windows_path(cls, path):
+        temp = path.replace(':', '').replace('\\', '/')
```

Comment:
Windows paths have a ':' and slashes in the wrong direction. Docker daemon won't allow this.